### PR TITLE
[DST-213]: Adding `tailwindcss-react-aria-components` plugin

### PR DESCRIPTION
**Pull request by OsamaAbdellateef**
We need to check all the component styles 😢 

- [ ] Accordion
- [ ] Autocomplete
- [ ] Badge
- [ ] Button
- [ ] Calendar
- [ ] Card
- [ ] Checkbox
- [ ] Chevron
- [ ] ComboBox
- [ ] DateField
- [ ] DatePicker
- [ ] Dialog
- [ ] Divider
- [ ] FieldBase
- [ ] Header
- [ ] Headline
- [ ] HelpText
- [ ] Input
- [ ] Label
- [ ] Link
- [ ] List
- [ ] ListBox
- [ ] Menu
- [ ] Message
- [ ] NumberField
- [ ] Overlay
- [ ] Provider
- [ ] Radio
- [ ] SearchField
- [ ] Select
- [ ] Slider
- [ ] Switch
- [ ] Table
- [ ] Tabs
- [ ] TagGroup
- [ ] Text
- [ ] TextArea
- [ ] TextField
- [ ] Tooltip
- [ ] XLoader

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
     "prettier": "3.0.3",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "tailwindcss-react-aria-components": "1.0.0-beta.1",
```
Comment (sebald) [inline package.json:33]: Should not be added to the root 😄

File: themes/theme-b2b/tailwind.config.ts [hunk 2]
```diff
@@ -3,6 +3,7 @@ import type { Config } from 'tailwindcss';
 import { preset } from './src/preset';
 
 export default {
+  plugins: [require('tailwindcss-react-aria-components')],
```
Comment (sebald) [inline themes/theme-b2b/tailwind.config.ts:6]: If we add it to our `createPreset` helper every theme will get it! :)

Comment (OsamaAbdellateef) [reply themes/theme-b2b/tailwind.config.ts:6]: now they are seperated

File: themes/theme-core/tailwind.config.ts [hunk 3]
```diff
@@ -5,6 +5,7 @@ import { preset } from './src/preset';
 // Figma File: https://www.figma.com/file/RiWJBV4Z8L8ycVvUuMYXbm/%F0%9F%93%93-CR---Components-2022?node-id=1452-1785&t=YaLGVHzniD5mOtbJ-0
 
 export default {
+  plugins: [require('tailwindcss-react-aria-components')],
```
Comment (sebald) [inline themes/theme-core/tailwind.config.ts:8]: Can't we add it to the preset? :)

**Review by sebald — CHANGES_REQUESTED** [review]

File: themes/theme-core/tailwind.config.ts (hunk 3 above)
Comment (sebald) [reply themes/theme-core/tailwind.config.ts:8]: Here:

https://github.com/marigold-ui/marigold/blob/main/packages/theme-preset/src/create-preset.ts#L17

**Comment by sebald** [comment]
@OsamaAbdellateef @sarahgm prefixed the plugin so we can merge it, can you check again if it breaking any CSS please :)

File: packages/theme-preset/src/create-preset.ts [hunk 4]
```diff
@@ -15,6 +17,7 @@ export const createPreset = (name: string, config: Partial<OptionalConfig>) =>
         },
       },
       plugins: [
+        racSelectorPlugin({ prefix: 'rac' }),
```
Comment (sarahgm) [inline packages/theme-preset/src/create-preset.ts:20]: how do we should use this than? @sebald

**Review by sarahgm — APPROVED** [review]

**Pull request merged**